common/Profanity.js

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const Filter = require('bad-words');
+
+const filter = new Filter();
+
+const includesProfanity = (s) => filter.isProfane(s);
+
+module.exports = {
+  includesProfanity,
+};
```

Comment:
Lowercase this file name to keep consistency in this directory.  Uppercase file names make it look like it should be a library.